internal/config/config.go

```diff
@@ -91,3 +109,15 @@ func GetConfigFile() string {
 
 	return filepath.Join(configFilePath, "config.yml")
 }
+
+func AccessTokenIsSet(filename string) (bool, error) {
+	config, err := NewConfigFromFile(filename, os.ReadFile)
+	if err != nil {
+		return false, err
+	}
+	if config.AccessToken != "" {
+		return false, errors.NewError("Your access token is already set. Remove it from the config if you wish to reset it.")
```

Comment:
just checking that the location of the config file is well known to all users?

Reply:
Yeah, it's [in the readme](https://github.com/launchdarkly/ldcli/blob/main/README.md#configuration).